modules/s2/redshift/requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy==1.16.6
```

Comment:
sNumpy is not needed here because it's installed by default in the Redshift clusters. Remember to update the shasum file after this change.

Reply:
Yep, I removed the dependency from the lib too so thanks for this one